client/post-editor/editor-drawer/index.jsx

```diff
@@ -326,6 +329,7 @@ const enhance = flow(
 			isJetpack: isJetpackSite( state, siteId ),
 			isSeoToolsModuleActive: isJetpackModuleActive( state, siteId, 'seo-tools' ),
 			jetpackVersionSupportsSeo: isJetpackMinimumVersion( state, siteId, '4.4-beta1' ),
+			jetpackVersionSupportsLocation: isJetpackMinimumVersion( state, siteId, '6.3-beta1' ),
```

Comment:
i'm running the Jetpack Beta Tester plugin and my version numbers look more like `6.3-beta-19269-98a6d34-branch-6.3`. Is that expected?

Reply:
Made the version number check more generic in a9703c5 so that it now matches against version numbers from the Jetpack Beta Tester plugin.